src/translations/default/about.mdx

```diff
@@ -0,0 +1,45 @@
+**CONTENT_NOT_TRANSLATED**
+
+Cela signifie que son code source peut être librement réutilisé et modifié, sous condition que ces modifications soient mises à disposition sous les mêmes conditions. Cela signifie également que les décisions d'évolution du logiciel seront progressivement prises de manière démocratique au fur et à mesure que la communauté qui le construit s'agrandira.
+
+** Nous recherchons des financements pour développer les usages et garantir la pérennité d'Open Terms Archive. Vous connaissez des moyens de nous aider ? Contactez-nous ! **
+
+### Équipe cœur
+
+L'équipe qui fabrique et fait grandir Open Terms Archive est incubée au sein du Bureau de l'Ambassadeur pour les affaires numériques.
+
+<Contributors type="core" alignX="left" />
+
+<Contributors subtitle="Alumnis" type="alumnis" alignX="left" className="mb__L" />
+
+##### Financeurs
+
+<ThumbGalery align="left" small={true}>
+  <ThumbGaleryItem src="/images/logo-meae.png" width="132" height="120" />
+  <ThumbGaleryItem src="/images/france-relance.png" width="77" height="77" />
+  <ThumbGaleryItem src="/images/nextgen.png" width="191" height="48" />
```

Comment:
Is this duplicated in every language then?! This is going to be a nightmare to maintain.

Reply:
I don't have a better solution for now and even if we have to change the page 3 times a year in 5 languages it doesn't seem so much a nightmare to me.